.node-version

```diff
@@ -0,0 +1,2 @@
+5
+
```

Comment:
The lint didn't complain about this, but I want to make sure this isn't an issue for you in the future. Make sure you delete this line, after you've set `ensure_newline_at_eof_on_save`.

You can tell if there isn't a newline at the EOF in the github diff through the 🚫  (or similar). Note how the files created by the yeoman generator don't have an extra blank line, but also don't have the 🚫 , but don't have a blank line either.

![github-diff-no-newline-warning](https://cloud.githubusercontent.com/assets/4823640/15558199/2e3008f2-228d-11e6-95f1-18f3e839fa67.png)